TTS/trainer.py

```diff
@@ -820,8 +820,12 @@ def _fit(self) -> None:
         """🏃 train -> evaluate -> test for the number of epochs."""
         if self.restore_step != 0 or self.args.best_path:
             print(" > Restoring best loss from " f"{os.path.basename(self.args.best_path)} ...")
-            self.best_loss = load_fsspec(self.args.restore_path, map_location="cpu")["model_loss"]
-            print(f" > Starting with loaded last best loss {self.best_loss}.")
+            fsspec = load_fsspec(self.args.restore_path, map_location="cpu")
```

Comment:
fsspec is a module we use for saving and loading. It does not make sense as the name here. Maybe use `checkpoint`

Reply:
done